internal/server/evaluation/evaluation.go

```diff
@@ -91,3 +112,56 @@ func (s *Server) Boolean(ctx context.Context, r *rpcevaluation.EvaluationRequest
 
 	return resp, nil
 }
+
+// Batch takes ina list of *evaluation.EvaluationRequest and returns their respective responses.
+func (s *Server) Batch(ctx context.Context, b *rpcevaluation.BatchEvaluationRequest) (*rpcevaluation.BatchEvaluationResponse, error) {
+	resp := &rpcevaluation.BatchEvaluationResponse{
+		Responses: []*rpcevaluation.EvaluationResponse{},
+	}
+
+	for _, req := range b.GetRequests() {
+		f, err := s.store.GetFlag(ctx, req.NamespaceKey, req.FlagKey)
+		if err != nil {
+			var errnf errs.ErrNotFound
+			if b.GetExcludeNotFound() && errors.As(err, &errnf) {
```

Comment:
```suggestion
			if b.GetExcludeNotFound() && errors.As(err, &errnf) {
```

we should remove this `excludeNotFound` from the proto, by default we're not going to early return if flag not found, we will simply map this to an error response type (as we talked about on zoom)